### PR TITLE
Add advanced settings UI

### LINKED ISSUE #5172: Implement Advanced Settings (Mailserver switching)
### Description

*Type*: Feature

*Summary*: user should be able to switch between mailservers.

#### Expected behavior
Design is [here](zpl.io/agveN81) 
image

#### Actual behavior
No advanced settings in Profile

### PR DESCRIPTION
**Pull request by vitvly**
Fix #5172 

UI note: the mailserver selection is implemented via view list instead of dropdowns, similarly to mobile version.

### CODE REVIEW

**Review by goranjovic — APPROVED** [review]

File: src/status_im/ui/screens/desktop/main/tabs/profile/styles.cljs [hunk 1]
```diff
@@ -9,14 +9,14 @@
...
    :margin-bottom 16})
 
-(def logout-row
+(def profile-row
   {:justify-content   :space-between
    :flex-direction    :row
    :margin-horizontal 24
    :align-self        :stretch
-   :margin-top        60})
+   :margin-top   40})
```
Comment (flexsurfer) [inline src/status_im/ui/screens/desktop/main/tabs/profile/styles.cljs]: indentation

File: src/status_im/ui/screens/desktop/main/tabs/profile/styles.cljs [hunk 2]
```diff
@@ -133,3 +134,18 @@
    :border-left-color  :transparent
    :border-right-color :transparent
    :border-top-color   colors/tooltip-green})
+
+(def advanced-settings-title
+  {:margin        24
+   :font-size     20
+   :font-weight  "600"})
```
Comment (flexsurfer) [inline src/status_im/ui/screens/desktop/main/tabs/profile/styles.cljs]: format

**Review by flexsurfer — APPROVED** [review]

**Comment by churik** [comment]
@siphiuel thanks for your PR, amazing stuff!
One small thing - can you please make the whole area of "Advanced options" clickable? 
For now clickable is only label, so if you click on red area, marked in screenshot, nothing happens.
whole_are

Also according to design when "Advanced settings" screen  is opened, text  should be highlighted and bold (it is not critical, so can add as separate issue and implement a bit later, if it takes too much time to implement). Tell me what you think.
grey

**Comment by vitvly** [comment]
@churik  Thanks for taking a look! Made requested changes.

**Comment by churik** [comment]
@siphiuel 
On IOS and Android builds "Mailservers settings" styles are changed: 
styles
There are no issues on desktop from my POV.

File: src/status_im/ui/screens/desktop/main/tabs/profile/styles.cljs (hunk 2 above)
Comment (vitvly) [reply src/status_im/ui/screens/desktop/main/tabs/profile/styles.cljs]: Fixed, thanks!

**Comment by vitvly** [comment]
Fixed mobile UI. Sorry for breaking things (

**Comment by maxhora** [comment]
@churik rebase conflict is resolved, thanks

**Comment by churik** [comment]
Tested on Mobile:
- installation
- account is created
- view of Mailserever in Profile
- mailserver switching

Tested on Desktop:
- installation;
- account is created;
- switching to other mailserverver;
- sending\receiving messages after mailserver switching;
- log in \ log out;
- 'Cancel' button when switching to other mailserver;

Known: 
'Advanced settings' label is cut when it is selected;
setti

 RN-related issue: https://github.com/status-im/status-react/issues/5113

**Pull request merged**